### PR TITLE
INVP-423 ModalStack docs

### PR DESCRIPTION
**Pull request by mpelzsherman**
[Chore](https://rentpath.atlassian.net/browse/INVP-423)

Added Markdown docs for the ModalStack component.

Also updated docs for working locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +88,7 @@ Examples
 | .SomeComponent-isActive | X | | X |
 | .SomeComponent_Navigation-next | X | X | X |
 | .SomeComponent_NavigationControls-invalidError | X | X | X |
+
+#### ModalStack
```
Comment (tadjohnston) [inline README.md]: I think this should probably go in a read me in `packages/react-ui-core` instead.  We should probably move the react-themed stuff there as well.

Comment (mpelzsherman) [reply README.md]: react-ui-core doesn't have it's own top-level README yet. Adding one just for this one link didn't makes sense to me, but if you prefer I can add one.

**Comment by mpelzsherman** [comment]
Sorry - forgot about `git cz`. I'll be sure to use it going forward.

**Comment by mpelzsherman** [comment]
Replaced by #457

**Pull request closed without merging**